client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -259,13 +259,13 @@ function CheckoutSummaryHelp() {
 				supportVariationDetermined && (
 					<CheckoutSummaryHelpButton onClick={ handleHelpButtonClicked }>
 						{ isSupportChatUser
-							? translate( 'Questions? {{underline}}Ask a Happiness Engineer.{{/underline}}', {
+							? translate( 'Questions? {{underline}}Ask a Happiness Engineer{{/underline}}', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.